src/state/pymMiddleware.js

```diff
@@ -0,0 +1,13 @@
+import pym from 'pym.js';
+
+const pymChild = new pym.Child();
+
+/**
+ * Update pym each time an action is fired.
+ */
+const pymMiddleware = () => next => (action) => {
+  pymChild.sendHeight();
+  return next(action);
+};
```

Comment:
I would doubt that it would be that laggy, but if it is we could debounce later. It occurred to me too, but I figured just adjusting the height of something isn't really laggy given how relatively rarely these events are fired.